src/main/java/redis/embedded/util/OSDetector.java

```diff
@@ -54,8 +54,11 @@ private static Architecture getUnixArchitecture() {
             input = new BufferedReader(new InputStreamReader(proc.getInputStream()));
             while ((line = input.readLine()) != null) {
                 if (line.length() > 0) {
-                    if (line.contains("64")) {
-                        return Architecture.x86_64;
+		    if (line.contains("64")) {
+                        if (line.contains("aarch"))
+                             return Architecture.aarch64;
```

Comment:
Give brackets as they have given previously for all if statements.

Reply:
as you can see the line which i have replced it with here they have not used the braces should i give it here